src/adjoint_sensitivity.jl

```diff
@@ -161,7 +162,29 @@ function (S::ODEAdjointSensitivityFunction)(du,u,p,t)
         vec(f(u, p, t))
       end
     end
-    dλ[:], dgrad[:] = back(λ)
+
+    _idy, iback = Zygote.pullback(S.sol.prob.p) do
```

Comment:
why is this call separate from line 156? Won't this be computed simultaneously to the other?